mdata/aggmetric.go

```diff
@@ -410,7 +408,7 @@ func (a *AggMetric) persist(pos int) {
 	}
 
 	// Every chunk with a T0 <= this chunks' T0 is now either saved, or in the writeQueue.
-	a.lastSaveStart = chunk.Series.T0
+	atomic.StoreUint32(&a.lastSaveStart, chunk.Series.T0)
```

Comment:
similar to my comment in SyncChunkSaveState, we need to account for others setting higher values using a swap loop.